<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-26T10:46:59Z" name="demoFrameworkV1" finished-at="2023-05-26T10:47:25Z" duration-ms="25941">
    <groups>
    </groups>
    <test started-at="2023-05-26T10:46:59Z" name="Banking Login Test" finished-at="2023-05-26T10:47:25Z" duration-ms="25941">
      <class name="com.demoFramework_v1.testCases.TC_LoginTestDDT_01">
        <test-method is-config="true" signature="SetUp(java.lang.String)[pri:0, instance:com.demoFramework_v1.testCases.TC_LoginTestDDT_01@f381794]" started-at="2023-05-26T10:46:59Z" name="SetUp" finished-at="2023-05-26T10:47:04Z" duration-ms="4523" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.demoFramework_v1.testCases.TC_LoginTestDDT_01@f381794]" started-at="2023-05-26T10:47:05Z" name="loginDDT" data-provider="LoginData" finished-at="2023-05-26T10:47:18Z" duration-ms="13259" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr502834]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[EzemAqE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.demoFramework_v1.testCases.TC_LoginTestDDT_01@f381794]" started-at="2023-05-26T10:47:18Z" name="loginDDT" data-provider="LoginData" finished-at="2023-05-26T10:47:24Z" duration-ms="6054" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[lmngr502835]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ytrgkk6]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.demoFramework_v1.testCases.TC_LoginTestDDT_01.loginDDT(TC_LoginTestDDT_01.java:34)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:com.demoFramework_v1.testCases.TC_LoginTestDDT_01@f381794]" started-at="2023-05-26T10:47:24Z" name="TearDown" finished-at="2023-05-26T10:47:25Z" duration-ms="795" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.demoFramework_v1.testCases.TC_LoginTestDDT_01 -->
    </test> <!-- Banking Login Test -->
  </suite> <!-- demoFrameworkV1 -->
</testng-results>
